WordPress/gutenberg
# Use Badge component in page markers

## What?


Partial to https://github.com/WordPress/gutenberg/issues/68020
Alternative to https://github.com/WordPress/gutenberg/pull/68086

Use the new Badge component for the page markers.


## Why?


To improve consistency and sustainability.

## How?

Using the new component and removing redundant styles.

## Testing Instructions

1. Set up the front page and blog page for your site.
2. Open the site editor
3. Go to the "Pages" section, confirm the pages that were chosen for the front page and blog page are using the Badge component to indicate this.
4. Edit the front page, and blog page, and confirm that the marker in the sidebar is also using the component.

### Testing Instructions for Keyboard


## Screenshots or screencast 

https://github.com/user-attachments/assets/bcc41a03-88d3-4a6d-8027-4fe4b925c5b9
